client/lobbies/selected-race.jsx

```diff
@@ -6,7 +6,7 @@ import { RaceButton, StyledRaceIcon } from './race-picker'
 
 import { colorDividers } from '../styles/colors'
 
-const Deselected = styled.span`
+export const Deselected = styled.span`
```

Comment:
nit: if we're gonna be using this for other purposes, might be good to rename it, 'Deselected' doesn't make much sense for the new `disabledRace` case